lib/hets/prove_options.rb

```diff
@@ -37,6 +37,8 @@ def prepare_sentences(field)
         @options[field].map! do |sentence|
           if sentence.is_a?(Sentence)
             sentence.name
+          else
+            sentence
```

Comment:
this is a bad variable name, because if the object is not a `Sentence`, it is returned as `sentence`.